api/docs/docs/access.mdx

```diff
@@ -3,95 +3,108 @@ id: access
 title: Accessing the API
 description: Register for the Covid Act Now API.
 ---
-import SignupForm from '@site/src/components/SignupForm';
 
-## Getting Started
+import SignupForm from "@site/src/components/SignupForm";
 
-<SignupForm />
+## Sign up
 
+<SignupForm />
 
-## Examples
+## Getting started
 
 In the examples below, replace `YOUR_KEY_HERE` with the API key from Step 1 above.
 
-### State summary data CSV
+The Covid Act Now API has coverage for states, counties, and metro areas (CBSAs). You can query the data in CSV or JSON formats.
 
-A CSV file containing the latest data and metrics for all states.
+Learn how to query:
 
-```
-https://api.covidactnow.org/v2/states.csv?apiKey=YOUR_KEY_HERE
-```
+- **[Current values for all states, counties, and metros](#current-values-for-all-states-counties-or-metros)**
+- **[Historic values for all states, counties, and metros](#historic-values-for-all-states-counties-or-metros)**
+- **[Current and historic values for an individual state, county, or metro area](#individual-locations)**
+
+### Current values for all states, counties, or metros
 
-[More details](/api#tag/State-Data/paths/~1states.csv?apiKey={apiKey}/get)
+These endpoints contain the latest available data for all states, counties, and cbsas (metros).
 
-### County summary data CSV
+Each file contains all locations in the region type.
+For example, `https://api.covidactnow.org/v2/counties.csv?apiKey=YOUR_KEY_HERE` contains all US counties.
 
-A CSV file containing the latest data and metrics for all counties.
+#### CSV format
 
 ```
+https://api.covidactnow.org/v2/states.csv?apiKey=YOUR_KEY_HERE
 https://api.covidactnow.org/v2/counties.csv?apiKey=YOUR_KEY_HERE
+https://api.covidactnow.org/v2/cbsas.csv?apiKey=YOUR_KEY_HERE
 ```
 
-[More details](/api#tag/County-Data/paths/~1counties.csv?apiKey={apiKey}/get)
+Read the API Reference for [/v2/states.csv](/api/#tag/State-Data/paths/~1states.csv?apiKey={apiKey}/get), [/v2/counties.csv](/api#tag/County-Data/paths/~1counties.csv?apiKey={apiKey}/get), [/v2/cbsas.csv](/api#tag/CBSA-Data/paths/~1cbsa~1{cbsa_code}.json?apiKey={apiKey}/get).
 
-### Historical data for all states CSV
-
-A CSV file containing the latest data and metrics for all states.
+#### JSON format
 
 ```
-https://api.covidactnow.org/v2/states.timeseries.csv?apiKey=YOUR_KEY_HERE
+https://api.covidactnow.org/v2/states.json?apiKey=YOUR_KEY_HERE
+https://api.covidactnow.org/v2/counties.json?apiKey=YOUR_KEY_HERE
+https://api.covidactnow.org/v2/cbsas.json?apiKey=YOUR_KEY_HERE
 ```
 
-[More details](/api#tag/State-Data/paths/~1states.timeseries.csv?apiKey={apiKey}/get)
+### Historic values for all states, counties, or metros
 
-### Historical data for all counties CSV
+These files contain the entire history for all states, counties, and cbsas (metros).
```

Comment:
To help readers understand that we mean a timeseries, not a history of timeseries versions:

`.... the entire history (a timeseries per field) for all ...`